2022-06-09-Austin/readme.md

```diff
@@ -0,0 +1,51 @@
+          <li>Project contributors won't be able to help you with individual bugs or issues</li>
+          <li>Get an introductory understanding of the <a href="https://github.com/openjs-foundation/summit/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Austin+2020%22">technical topics</a> that are being discussed. With the exception of the content on New Contributor Day, these topics will not be explained.</li>
+        </ul>
+        <h3>How to prepare for the event:</h3>
+        <ul>
+          <li>Check out the <a href="https://github.com/openjs-foundation/summit/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Austin+2020%22">summit sessions</a></li>
+          <li>Give a 👍 for sessions you would like to participate in</li>
+          <li>Read up on technical topics that you are not familiar with</li>
+        </ul>
+    </td>
+    <td width="50%" style="padding: 0 0 0 50px; vertical-align: top;">
+       <h2>Maintainer or core contributor of an <a href="https://openjsf.org/projects/">OpenJS Project</a></h2>
+        This is a great opportunity to get together and be productive. Use the time to talk about important project topics, pair with each other, resolve bugs, build consensus, or tinker on new ideas. It's also a great time to collaborate with peers from other projects and learn more about what they are doing and how the OpenJS Foundation can support your work.
+        <h3>How to prepare for the event:</h3>
+        <ul>
+          <li>Speak with your project community and evaluate topics that need a broader forum to talk about</li>
+          <li>Create a <a href="https://github.com/openjs-foundation/summit/issues/new?assignees=christian-bromann%2C+jorydotcom%2C+keywordnew%2C+wesleytodd%2C+waleedashraf&labels=Austin+2020%2C+Collaborator+Summit%2C+Session+Proposal&template=summit-topic-proposal.md&title=Session+Proposal%3A+">session proposal</a> and fill out the correspondig issue form</li>
+          <li>If you are a session facilitator let the organizing team know about specific session requirements by reaching out in the #collabsummit Slack channel (e.g. whiteboard, markers, post-its etc.)</li>
+          <li>Read our <a href="https://github.com/openjs-foundation/summit/blob/HEAD/SESSION_FACILITATOR_GUIDE.md">Session Facilitator Guide</a> for tips and best practices on running your session, and reach out to the organizers for help.
+        </ul>
+    </td>
+  </tr>
+</table>
+
+## Important for all participants ☝
+
+We want everyone, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age, religion or lack thereof, to feel welcome and safe and collaborate with each other in a harassment free environment. We do not tolerate harassment of conference participants in any form. Sexual language and imagery is not appropriate for any conference venue, including talks. Summit participants violating these rules may be sanctioned or expelled.
```

Comment:
We should probably mention and link to the Code of Conduct here.

Reply:
```suggestion
The [Linux Foundation Events Code of Conduct](https://events.linuxfoundation.org/about/code-of-conduct/) applies to all event spaces.

We want everyone, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age, religion or lack thereof, to feel welcome and safe and collaborate with each other in a harassment free environment. We do not tolerate harassment of conference participants in any form. Sexual language and imagery is not appropriate for any conference venue, including talks. Summit participants violating these rules may be sanctioned or expelled.
```